engine/src/main/java/org/hibernate/validator/constraintvalidation/HibernateConstraintValidatorInitializationContext.java

```diff
@@ -55,4 +56,12 @@
 	 * @return the tolerance
 	 */
 	Duration getTemporalValidationTolerance();
+
+	/**
+	 * @param type The type of payload to retrieve
+	 * @return an instance of the specified type set by the user via
```

Comment:
Please make the long version the description of the method and keep a short one for the `@return`.